posts/flat-file-exports/flat-files-compare.qmd

```diff
@@ -0,0 +1,440 @@
+
+```
+
+```
+  fov cell_ID A1BG ... ZZZ3 Negative1 ... Negative9 SystemControl1 ... SystemControl99
+  1       1    0 ...    0         0 ...         0              0 ...               0
+  1       2    0 ...    0         0 ...         0              0 ...               0
+  1       3    0 ...    0         0 ...         0              0 ...               0
+  1       4    1 ...    0         0 ...         0              0 ...               0
+  1       5    0 ...    0         0 ...         0              0 ...               0
+  1       6    0 ...    0         0 ...         0              0 ...               0
+  1       7    0 ...    0         0 ...         0              0 ...               0
+  1       8    0 ...    0         0 ...         0              0 ...               0
+  1       9    1 ...    0         0 ...         0              0 ...               0
+  1      10    0 ...    0         0 ...         0              0 ...               0
+```
+
+:::
+
+# Metadata file
+
+The primary difference between legacy and AtoMx formats for the metadata is that the latter includes processed results (_i.e._, extra columns).
+
+::: {.panel-tabset group="version"}
+
+## Legacy
+
+```{r}
+#| eval: false
+library(plyr)
+library(dplyr)
+meta_file_old <- read.csv("path/to/NSCLC/Lung5_Rep1/Lung5_Rep1-Flat_files_and_images/Lung5_Rep1_metadata_file.csv", header=TRUE)
+head(meta_file_old) %>% str
+```
+
+```
+'data.frame':	6 obs. of  20 variables:
+ $ fov               : int  1 1 1 1 1 1
+ $ cell_ID           : int  1 2 3 4 5 6
+ $ Area              : int  1259 3723 2010 3358 1213 2647
+ $ AspectRatio       : num  1.34 1.45 1.62 0.47 1 1.38
+ $ CenterX_local_px  : int  1027 2904 4026 4230 4258 66
+ $ CenterY_local_px  : int  3631 3618 3627 3597 3629 3622
+ $ CenterX_global_px : num  4216 6093 7215 7419 7447 ...
+ $ CenterY_global_px : num  158848 158835 158844 158814 158846 ...
+ $ Width             : int  47 87 68 48 38 72
+ $ Height            : int  35 60 42 102 38 52
+ $ Mean.MembraneStain: int  3473 3895 2892 6189 8138 5713
+ $ Max.MembraneStain : int  7354 13832 6048 16091 19281 12617
+ $ Mean.PanCK        : int  715 18374 3265 485 549 1220
+ $ Max.PanCK         : int  5755 53158 37522 964 874 5107
+ $ Mean.CD45         : int  361 260 378 679 566 433
+ $ Max.CD45          : int  845 1232 908 2322 1242 957
+ $ Mean.CD3          : int  22 13 19 5 17 11
+ $ Max.CD3           : int  731 686 654 582 674 547
+ $ Mean.DAPI         : int  4979 1110 10482 6065 3311 4151
+ $ Max.DAPI          : int  26374 13229 33824 39512 30136 19269
+```
+
+
+## AtoMx 1.3.2
+
+
+```{r}
+#| eval: false
+library(plyr)
+library(dplyr)
+meta_file_new <- read.csv("path/to/breast_cancer_example/flatFiles/AUG29_13INTEGR_6K_BRST_PS_S2/AUG29_13INTEGR_6K_BRST_PS_S2_metadata_file.csv.gz", header=TRUE)
+head(meta_file_new) %>% str
+```
+
+
+```
+'data.frame':	6 obs. of  68 variables:
```

Comment:
Is this one standard? Appears to have two cell typing results plus some columns that start with 'i.'

Reply:
The file comes from the export module but it has analysis results added to it (which is standard for the export module to do). But it is not directly from "raw" data so there are additional columns added to it, if that makes sense. 

I will adjust this section to clarify.

Reply:
I have updated the post with tables that describe the columns.

1a0416d378b30a2f45278643629781e3b497fd92